### ISSUE TITLE
Support skipImports

### ISSUE THREAD

**Issue by josejulio** [juusaw/ts-to-io#2]
My types are spread into multiple files, say I have A.ts and B.ts.
A.ts imports B.ts

So if I run `ts-to-io A.ts > ACodecs.ts` my file has all the definitions found in B.ts.

Even if B.ts is needed and some of its code needs to be duplicated into the `ACodecs.ts` file, I would like to avoid the unneeded definitions (i.e. interfaces found in `B.ts`).

For example, if I have `A.ts` as:
```typescript
import { DataB } from './B';

export interface DataA {
  abc: DataB;
}
```

and `B.ts` as:
```typescript
export interface DataB {
  def: string;
}
```

I would like `yarn ts-to-io A.ts --skipImports` to spit out:

```typescript
import * as t from "io-ts"

const DataA = t.type({abc: t.type({def: t.string})})
```

instead of 
```typescript
import * as t from "io-ts"

const DataB = t.type({def: t.string})

const DataA = t.type({abc: t.type({def: t.string})})
```

**Comment by juusaw** [comment]
Hi, I'm currently implementing different configuration options in #1. Skipping imported declarations is actually one of them so you should be able to do this some time in the (near) future!

**Comment by josejulio** [comment]
I saw the PR literally after finishing this request. Awesome :)

**Comment by juusaw** [comment]
Fixed in #1. Skipping imports is now the default behavior (hence the minor version bump).
